script/dev

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This is a script for running Compose inside a Docker container. It's handy for
+# development.
+#
+# $ ln -s `pwd`/script/dev /usr/local/bin/docker-compose
+# $ cd /a/compose/project
+# $ docker-compose up
+# 
+
+set -e
+DIR="$(dirname "$(readlink "$0")")"/..
+docker build -t docker-compose $DIR
```

Comment:
Perhaps we should follow the same logic as `docker run`: attempt to run, check for the "No such image" error, build the image and try again.

For actual distribution, I agree - we should be pulling a tagged image from the registry.